project/Dependencies.scala

```diff
@@ -70,26 +67,24 @@ object Dependencies {
       exclude("org.apache.logging.log4j", "log4j-api")
       exclude("org.apache.logging.log4j", "log4j-core"),
 
-    excludeGuava("com.google.apis"     % "google-api-services-storage"      % "v1-rev20190910-1.30.3"),
     excludeGuava("com.google.apis"     % "google-api-services-pubsub"       % "v1-rev20191001-1.30.3"),
-    excludeGuava("com.google.auth"     % "google-auth-library-oauth2-http"  % "0.24.1"),
```

Comment:
google-api-services-storage and google-auth-library-oauth2-http are pulled in by workbench-libs google2, not necessary to list here.